src/gatt/gatt_properties.rs

```diff
@@ -95,30 +78,21 @@ macro_rules! _properties {
         }
     }
 }
-
 macro_rules! properties {
```

Comment:
The simplification helps, though IMO these macros are still a bit complicated for something that's only used twice. I'm curious how many lines are saved.